shell/browser/common_web_contents_delegate.cc

```diff
@@ -253,11 +253,11 @@ void CommonWebContentsDelegate::ResetManagedWebContents(bool async) {
     base::ThreadTaskRunnerHandle::Get()->PostNonNestableTask(
         FROM_HERE,
         base::BindOnce(
-            [](scoped_refptr<ElectronBrowserContext> browser_context,
+            [](ElectronBrowserContext* browser_context,
                std::unique_ptr<InspectableWebContents> web_contents) {
               web_contents.reset();
             },
-            base::RetainedRef(browser_context_), std::move(web_contents_)));
+            base::Unretained(browser_context_), std::move(web_contents_)));
```

Comment:
this should not be needed anymore https://github.com/electron/electron/commit/363aafc25b945567e6ab596edb52d319df2701cc, can you remove and see if crash happens.

```
base::ThreadTaskRunnerHandle::Get()->DeleteSoon(FROM_HERE,
                                                web_contents_.release());
```

Reply:
nice catch, updated.